examples/slider/slider-thermostat.html

```diff
@@ -0,0 +1,521 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Slider Examples with aria-orientation and aria-valuetext | WAI-ARIA Authoring Practices 1.2</title>
+
+  <!--  Core js and css shared by all examples; do not modify when using this template. -->
+  <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+  <link rel="stylesheet" href="../css/core.css">
+  <script src="../js/examples.js"></script>
+  <script src="../js/highlight.pack.js"></script>
+  <script src="../js/app.js"></script>
+
+  <link rel="stylesheet" href="css/slider-thermostat.css">
+  <script src="js/slider-thermostat.js"></script>
+</head>
+<body>
+  <nav aria-label="Related Links" class="feedback">
+    <ul>
+      <li><a href="../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/projects/3">Related Issues</a></li>
+      <li><a href="../../#slider">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h1>Slider Examples with aria-orientation and aria-valuetext</h1>
+    <div class="advisement">
+      <p><strong>WARNING!</strong> Some users of touch-based assistive technologies may experience difficulty utilizing widgets that implement this slider pattern because the gestures their assistive technology provides for operating sliders may not yet generate the necessary output.  To change the slider value, touch-based assistive technologies need to respond to user gestures for incrementing and decrementing the value by synthesizing key events.  This is a new convention that may not be fully implemented by some assistive technologies.  Authors should fully test slider widgets using assistive technologies on devices where touch is a primary input mechanism before considering incorporation into production systems.
+    </p>
+    </div>
+    <p>
+      The following example of a hypothetical online thermostat demonstrates the
+      <a href="../../#slider">slider design pattern.</a>
+      The desired temperature is set with the first slider, which is vertically oriented.
+      Two horizontal sliders set fan speed and mode.
```

Comment:
Since the fan design changed only to "speed", all "mode" info can be deleted. (There are a few more places to delete the word, "mode".) Btw, it was great idea to change the design to current one, @jongund and @patrickhlauke!

Reply:
@a11ydoer 
@mcking65 
The change to the reference links and descriptions you are requesting is  something that needs to be coordinated with Matt, since there are several pull requests related to slider examples pending.  When examples are merged to main the reference links in the other examples and the guidance document should be updated to reflect the current examples.